components/survey/element.js

```diff
@@ -208,17 +209,25 @@ export class MDNSurvey extends L10nMixin(LitElement) {
             @click=${this.#dismiss}
           ></mdn-button>
         </header>
-        <details ${ref(this._detailsRef)} @toggle=${this.#onToggle}>
-          <summary>${this._survey.question}</summary>
-          ${this._isOpen && this._source
-            ? html`
-                <iframe
-                  title=${ifDefined(this._survey.question)}
-                  src=${this._source}
-                ></iframe>
-              `
-            : nothing}
-        </details>
+        ${this._survey.link
+          ? html`<a
+              href=${this._source}
```

Comment:
Should probably mark this as an external link:
```suggestion
              class="external"
              href=${this._source}
```